.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - release-[1-9].*
```

Comment:
I still left this pattern even though strictly speaking for this repo it wouldn't be necessary. I felt we can align it once we have replace TC